README.md

```diff
@@ -13,128 +13,74 @@
-```
-
-And just like that you're started!
-
-## [BETA] SwiftUI
+Why show a quick start when we have an example app? Because it's so easy to get started, we can just drop in 2 code snippets and you're ready to go! This quick start uses Swift Package Manager and SwiftUI, but for other approaches, [see our installation instructions](https://wwt.github.io/SwiftCurrent/installation.html).
 
 ```swift
 .package(url: "https://github.com/wwt/SwiftCurrent.git", .upToNextMajor(from: "4.1.0")),
 ...
 .product(name: "SwiftCurrent", package: "SwiftCurrent"),
-.product(name: "BETA_SwiftCurrent_SwiftUI", package: "SwiftCurrent")
+.product(name: "SwiftCurrent_SwiftUI", package: "SwiftCurrent")
 ```
 Then make your first FlowRepresentable view:
 ```swift
 import SwiftCurrent
+import SwiftUI
 struct OptionalView: View, FlowRepresentable {
     weak var _workflowPointer: AnyFlowRepresentable?
     let input: String
     init(with args: String) { input = args }
     var body: some View { Text("Only shows up if no input") }
     func shouldLoad() -> Bool { input.isEmpty }
 }
-struct ExampleView: View, FlowRepresentable {
+struct ExampleView: View, PassthroughFlowRepresentable {
     weak var _workflowPointer: AnyFlowRepresentable?
     var body: some View { Text("This is ExampleView!") }
 }
 ```
-Then from your ContentView body, add: 
+Then from your `ContentView` or whatever view (or app) you'd like to contain the workflow, add the following view to the body: 
 ```swift
 import SwiftCurrent_SwiftUI
-...
-WorkflowLauncher(isLaunched: .constant(true), startingArgs: "Skip optional screen") {
-    thenProceed(with: OptionalView.self) {
-        thenProceed(with: ExampleView.self)
+// ...
+var body: some View { 
+    // ... other view code (if any)
+    WorkflowLauncher(isLaunched: .constant(true), startingArgs: "Skip optional screen") {
+        thenProceed(with: OptionalView.self) {
+            thenProceed(with: ExampleView.self)
+        }
     }
 }
 ```
 
-And just like that you're started!
-
-Here's an example of an app written with SwiftCurrent running wherever SwiftUI runs.  Check it out:
-
-https://user-images.githubusercontent.com/79471462/131555558-9e01e753-cafd-4ae2-af1b-b71a1aabb71f.mp4
-
+And just like that you've got a workflow! You can now add more items to it, or re-order the items that are there. To understand more of how this works and what is there [check out our developer docs](https://wwt.github.io/SwiftCurrent/How%20to%20use%20SwiftCurrent%20with%20SwiftUI.html)
 
-# Deep Dive
+# Check out our example apps
+We have [example apps](https://github.com/wwt/SwiftCurrent/tree/main/ExampleApps) for both SwiftUI and UIKit that show SwiftCurrent in action. They're even tested so you can see what it's like to test SwiftCurrent code. To run it locally, start by cloning the repo, open `SwiftCurrent.xcworkspace` and then run the `SwiftUIExample` scheme or the `UIKitExample` scheme. 
 
+# [See this to learn more](https://wwt.github.io/SwiftCurrent/index.html)
```

Comment:
This linking back to the README feels odd - link to a more specific page or update the text?

```suggestion
# [Learn more in our Docs](https://wwt.github.io/SwiftCurrent/index.html)
```